cli/cmd/lql.go

```diff
@@ -269,6 +279,18 @@ func runQuery(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, msg)
 	}
 
+	if lqlCmdState.Range != "" {
+		cli.Log.Debugw("retrieving natural time range")
+
+		var start, end time.Time
+		start, end, err = lwtime.ParseNatural(lqlCmdState.Range)
+		if err != nil {
+			return errors.Wrap(err, msg)
+		}
+		lqlCmdState.Start = start.UTC().Format(time.RFC3339)
+		lqlCmdState.End = end.UTC().Format(time.RFC3339)
```

Comment:
why do we need to do this conversion and formatting here? 🤔 Why not the start and end times get back from the new `lwtime` package ready to be used?

Reply:
This is very astute of you.  I believe `lwtime.ParseNatural` is doing the right thing here by returning go `time.Time` for start/end.  However, the API for `run` is expecting  a string (hence the conversion)...